examples/react-app/src/index.tsx

```diff
@@ -76,18 +78,28 @@ ReactDOM.render(
           display: 'flex',
           flexDirection: 'row',
           justifyContent: 'space-between',
+          alignItems: 'center',
         }}
       >
         <span>Menu Item</span>
+
         <PlmgSeparator
           thickness="thick"
+          style={{ alignSelf: 'stretch' }}
```

Comment:
Why did you add this style? Is it required for the component to work? If it is you should make a bug report so it is fixed for all implementations rather than only in this app.